.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Build
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
is there a reason why it does not run on `windows` and `mac-os`? it is possible to create a matrix for os too, see example [here](https://github.com/EcoExtreML/zampy/blob/26beebb53bda579e9bdac67fd12fbd5700165954/.github/workflows/build.yml#L19).